deepface/detectors/FastMtcnnWrapper.py

```diff
@@ -0,0 +1,57 @@
+import cv2
+from deepface.detectors import FaceDetector
+
+# Link -> https://github.com/timesler/facenet-pytorch
+# Examples https://www.kaggle.com/timesler/guide-to-mtcnn-in-facenet-pytorch
+
+def build_model():
+    # Optional dependency
+    try:
+        from facenet_pytorch import MTCNN as fast_mtcnn
+    except ModuleNotFoundError:
+        print("This is an optional detector, ensure the library is installed. \
```

Comment:
please raise an error instead of print. Otherwise, you will have an exception in line 16.

```python
raise ValueError("This is an optional detector...")
```

Reply:
should I raise ImportError instead?

```
except ModuleNotFoundError as e:
        raise ImportError("This is an optional detector, ensure the library is installed. \
              Please install using 'pip install facenet-pytorch' ") from e
```

Reply:
sure, that is better